UPGRADE-5.0.md

```diff
@@ -335,6 +335,11 @@ Process
    $process = Process::fromShellCommandline('ls -l');
    ```
 
+PropertyAccess
+--------------
+
+ * Removed support of passing `null` as 2nd argument of `PropertyAccessor::createCache()` method (`$defaultLifetime`), pass `0` instead.
```

Comment:
This should also be done in #32269 (we always document the deprecation and its removal in the same branch).

Reply:
Done in a71ee27180